docs/book/src/cronjob-tutorial/testdata/project/controllers/cronjob_controller.go

```diff
@@ -41,10 +41,9 @@ import (
 )
 
 /*
-Next, we'll need a few more bits in our Reconciler:
-
-We'll need the Scheme, in order to call some helpers that set owner references,
-and we'll need a Clock, which will allow us to fake timing in our tests.
+Next, kubebuilder has scaffolded a basic reconciler struct for us.
```

Comment:
This paragraph was ment for `docs/book/src/cronjob-tutorial/testdata/emptycontroller.go` not for `docs/book/src/cronjob-tutorial/testdata/project/controllers/cronjob_controller.go` and docs/book/src/multiversion-tutorial/testdata/project/controllers/cronjob_controller.go`.